modules/sc-mesh-secure-deployment/src/server-mesh.py

```diff
@@ -87,12 +87,25 @@ def add_message(uuid):
         return 'Not Valid Certificate'
 
 
+def verify_certificate(old, new):
+    """
+    Here we are validating the hash of the certificate. This is giving us the integrity of the file, not if the
+    certificate is valid. To validate if the certificate is valid, we need to verify the features of the certificate
+    such as NotBefore, notAfter, crl file and its signature, issuer validity, if chain is there then this for all but
+    for this we need to use x509 certificates.
+
+    """
+    old_md5 = hashlib.md5(old.read()).hexdigest()
+    new_md5 = hashlib.md5(new).hexdigest()
+    return old_md5 == new_md5
+
 
 def add_default_route(ip_gateway):
     inter = netifaces.interfaces()
     for interf in inter:
         # TODO: what it if doesn't start with wlan???
-        if interf.startswith('wlan'):
+        #if interf.startswith('wlan'):
```

Comment:
So I had also thought that we should do this, but does this then not introduce a problem if there's two interfaces that begin with "wl" (or if two interfaces begin with "wlan" we will also have the same issue)? Should we make it specifiable from the user?

Reply:
Right, I will leave it as 'wlan', so then we can think of another solution.